### ISSUE TITLE
[Bug] Login Page - LoginButtonFontColor not working

### ISSUE THREAD

**Issue by e1ze** [ironmansoftware/universal-dashboard#1479]
#### Describe the Issue
The parameter `-LoginButtonFontColor` on the `New-UDLoginPage` cmdlet is not working as intended. The color is not used, only the standard color is used.

#### To Reproduce
Just create a sample dashboard and change the LoginButtonFontColor, see script sample below

#### Expected behavior
The LoginButtonFontColor should the color defined

#### Screenshots
![grafik](https://user-images.githubusercontent.com/6794362/73051901-1bb9fb80-3e84-11ea-962c-351952089b2b.png)

#### Script Sample
My example (The Authentication method is based on a custom function for validating AD credentials)

```powershell
$LoginPage = New-UDLoginPage -AuthenticationMethod $AuthenticationMethod -Logo $logo -PageBackgroundColor $Cache:Config.Theme.LoginBGColor -Title $Cache:Config.Theme.LoginTitle -LoginFormBackgroundColor $Cache:Config.Theme.LoginFormBGColor -LoginFormFontColor $Cache:Config.Theme.LoginFormFontColor -LoginButtonBackgroundColor $Cache:Config.Theme.LoginButtonBGColor -LoginButtonFontColor $Cache:Config.Theme.LoginButtonFontColor -WelcomeText $Cache:Config.Theme.WelcomeText -LoadingText $Cache:Config.Theme.LoginLoadingText
```

The config is located in a powershell data file, f.e.:

```powershell
@{
  'Theme'           = @{
    LoginBGColor         = "#003452"
    LoginFormBGColor     = "#003452"
    LoginFormFontColor   = 'white'
    LoginButtonBGColor   = '#003452'
    LoginButtonFontColor = 'white'
    LoginTitle           = "Internal Dashboard"
    LoginLoadingText     = "Please wait while logging you in..."
    LoginErrorText       = "Invalid username or password, please try again."
    WelcomeText          = ""
    LogoPath             = "images\logo_transparent.png"
    LogoLoginHeight      = 114
    LogoLoginWidth       = 594
  }
{

(The logo is generated using new-udimage)

```

#### Version Information
- **Operating System**: Windows 10 1909 / Windows Server 2019 LTSB
- **PowerShell Version**: 5.1, 6.2.3, 7.0 RC2
- **Universal Dashboard Version**: 2.8.0, 2.8.1, 2.8.2
- **UD Hosting Method**: Powershell, IIS

#### Additional context
Another question. Is it possible to change the Login Page Background to an image or customize the login page via CSS in general?

thanks in advance,
René

**Comment by e1ze** [comment]
@adamdriscoll 
i just played around with the source code of the login page. It seems that the style part in the **href** from the login button is missing:

**Before:**

```html
<div class="card-action">
   <a href="#!">Login</a></div>
```

**After adding the style part:**

```html
<div class="card-action">
  <a href="#!" style="background: rgb(0, 52, 82) none repeat scroll 0% 0%; color: rgb(255, 255, 255);">Login</a></div>
```

Result:

![image](https://user-images.githubusercontent.com/6794362/73267958-556b6900-41da-11ea-86af-b5b64c965a8b.png)

**Comment by adamdriscoll** [comment]
This will be fixed in 2.9. I've also added login-page and login-card CSS class names so you can customize elements of the UD login page via CSS in the theme.

**Comment by adamdriscoll** [comment]
This issue has been mentioned on **Ironman Software Forums**. There might be relevant details there:

https://forums.universaldashboard.io/t/custom-login-page/2039/4
